packages/mui-material/src/styles/responsiveFontSizes.d.ts

```diff
@@ -1,12 +1,12 @@
 import { Breakpoint } from '@mui/system';
+import { TypographyProps } from '../Typography';
 import { Theme } from './createTheme';
-import { Variant } from './createTypography';
 
 export interface ResponsiveFontSizesOptions {
   breakpoints?: Breakpoint[];
   disableAlign?: boolean;
   factor?: number;
-  variants?: Variant[];
+  variants?: Array<TypographyProps['variant']>;
```

Comment:
Should not this come from `createTypography`?

```js
import { Typography } from './createTypography';

...
variants?: Array<keyof Typography>
```

I think this is better than using the type from the component since it is about theming.

Reply:
@siriwatknp Agree. Made the change.